tests/platform_tests/sfp/test_sfputil.py

```diff
@@ -49,6 +57,105 @@ def test_check_sfputil_presence(duthosts, enum_rand_one_per_hwsku_frontend_hostn
             assert parsed_presence[intf] == "Present", "Interface presence is not 'Present'"
 
 
+def get_namespace_cmd_option(duthost, asic_index):
+    """Get the namespace option used in the command"""
+    namespace = duthost.get_namespace_from_asic_id(asic_index)
+    return "-n {}".format(namespace) if namespace else ""
+
+
+def get_down_ports(duthost, ports, expect_up=True):
+    """Check and return the down ports among the given ports."""
+    ports_down = duthost.interface_facts(up_ports=ports)["ansible_facts"][
+        "ansible_interface_link_down_ports"]
+    db_ports_down = duthost.show_interface(command="status", up_ports=ports)["ansible_facts"][
```

Comment:
@longhuan-cisco Can you please explain the difference between ports_down and db_ports_down (wondering why db_ports_down is not sufficient)?

Reply:
Updated to only rely on db_ports_down